cpp/src_prims/sparse/distance/csr_spmv.cuh

```diff
@@ -0,0 +1,355 @@
+/*
```

Comment:
Do we have tests for these SPMV prims? Can we compare to reference dense SPMV or CPU equivalents?

Reply:
I've been testing the spmv distances through python by varying the data sizes (e.g. using more than 12000 cols to force the csr semiring, etc...).